com.unity.render-pipelines.high-definition/Documentation~/Dynamic-Resolution.md

```diff
@@ -18,16 +22,18 @@ To use dynamic resolution in your Project, you must enable dynamic resolution in
 1. In the Inspector for your HDRP Asset, go to **Rendering** **> Dynamic Resolution** and enable the **Enable** checkbox. For information on how to customize the rest of the HDRP Asset’s global dynamic resolution properties, see the dynamic resolution section of the [HDRP Asset documentation](HDRP-Asset.md#DynamicResolution).
 2. For every [Camera](HDRP-Camera.md) you want to perform dynamic resolution, go to the **General** section and enable **Allow Dynamic Resolution**.
 
-Dynamic resolution is not automatic, so you need to manually call the `DynamicResolutionHandler.SetDynamicResScaler(PerformDynamicRes scaler, DynamicResScalePolicyType scalerType)` function.
+HDRP does not use dynamic resolution automatically. To use dynamic resolution in your scene, call the following function:
```

Comment:
"HDRP does not perform the scaling for dynamic resolution" rather than that.

Dynamic res is enabled in the asset etc. Technically it can be  used even when no scaling happens.

Reply:
Thanks for this clarification!